api/controllers/projectGroupV2.js

```diff
@@ -0,0 +1,303 @@
+// Imports
+const defaultLog = require('winston').loggers.get('default');
+const Actions    = require('../helpers/actions');
+const projectDAO = require('../dao/projectDAO');
+const projectGroupDAO = require('../dao/projectGroupDAO');
+
+// Constants
+const PUBLIC_ROLES = ['public'];
```

Comment:
Can we create a constants file (similar to admin/public) and move these in there?

Reply:
Yep, good idea. Fixed